### ISSUE TITLE
Validation vs Test Set

### ISSUE THREAD

**Issue by fordaj** [fordaj/lowpy#37]
Decide whether to split some of the training data into a validation set
